scripts/core/superdesk-api.d.ts

```diff
@@ -319,6 +329,8 @@ declare module 'superdesk-api' {
         lock_session: any;
         rewritten_by?: IArticle['_id'];
         rewrite_of?: IArticle['_id'];
+        correction_by?: IArticle['_id'];
+        corrected_of?: IArticle['_id'];
```

Comment:
it would be more correct to name it `correction_by`/`correction_of`

Reply:
Petr and I are discussing whether we need these fields or not, if yes then I'll for sure rename them.